.github/workflows/main.yml

```diff
@@ -138,6 +138,23 @@ jobs:
           --runtime ${{ matrix.rid }}
           --self-contained
 
+      - name: Generate macOS .app bundle resources
+        if: ${{ startsWith(matrix.rid, 'osx-') && matrix.app == 'DiscordChatExporter.Gui' }}
+        shell: pwsh
+        run: >
+          ./bundle-macos-app.ps1
+          -MatrixAssetName "${{ matrix.asset }}"
+          -MatrixAppPath "${{ matrix.app }}"
+          -PublishDir "${{ matrix.app }}/bin/publish/"
+          -Version "${{ github.ref_type == 'tag' && github.ref_name || '999.9.9'}}"
+          -GitHubSha "${{ github.sha }}"
+          -GitHubRefType "${{ github.ref_type }}"
+          -GitHubRefName "${{ github.ref_name }}"
+          -BundleIdentifier "com.Tyrrrz.${{ matrix.asset }}"
+          -SpokenName "Discord Chat Exporter"
+          -Copyright "© Oleksii Holub"
+          -IconPath "favicon.icns"
```

Comment:
These last 4 parameters could probably just be hard-codded instead since they don't really need to be parametrized by the CI/CD job